README.md

```diff
@@ -172,6 +172,10 @@ Mbed OS has two options for the file system:
   The little file system (LittleFS) is a fail-safe file system we designed
   for embedded systems, specifically for microcontrollers that use flash
   storage.
+  
+  ``` cpp
+  LittleFileSystem fs("fs");
+  ```
```

Comment:
Added some code snippets to help users who are interchanging the different technology layers.

@AnotherButler, how do you want these bullet points formatted?

Reply:
This is fine.